components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/IdentityRecoveryConstants.java

```diff
@@ -388,13 +395,15 @@ public String toString() {
         public static final String SELF_REGISTRATION_SMSOTP_VERIFICATION_CODE_EXPIRY_TIME =
                 "SelfRegistration.VerificationCode.SMSOTP.ExpiryTime";
 
-        public static final String ENABLE_EMIL_VERIFICATION = "EmailVerification.Enable";
+        public static final String ENABLE_EMAIL_VERIFICATION = "EmailVerification.Enable";
```

Comment:
We cannot change the name of a public variable since it might break the functionality